packages/espressocash_app/lib/features/activities/widgets/off_ramp_tile.dart

```diff
@@ -22,16 +21,7 @@ class OffRampTile extends StatelessWidget {
   Widget build(BuildContext context) => OffRampOrderDetails(
         orderId: activity.id,
         builder: (context, order) => CpActivityTile(
-          title: order?.amount.let(
-                (amount) => context.l10n.activities_lblSoldToken(
-                  amount.format(
-                    context.locale,
-                    maxDecimals: 2,
-                    roundInteger: true,
-                  ),
-                ),
-              ) ??
-              '',
+          title: context.l10n.ramp_btnCashOut,
```

Comment:
`btn` messages should not be used here, they are call-to-actions, here it's rather a description of action that was already taken. It can be different in some languages.